docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/delegate-to-an-operator.md

```diff
@@ -1,16 +1,21 @@
+---
+sidebar_position: 1
+title: Delegate to an Operator
+---
+
 # Delegate to an Operator
 
-Follow these steps to initiate delegation of your restaked balance to an Operator. Delegation is an all or nothing operation. You must delegate all of your available Restaked balance to a single Operator.
+Follow the steps below to initiate delegation of your restaked balance to an Operator. Delegation is an all or nothing operation. You must delegate all of your available Restaked balance to a single Operator.
 
-**Step 1:** Navigate to the Operator tab to view a list of available Operators.
+**Step 1:** Navigate to the **Operator** page to view a list of available Operators.
```

Comment:
can / should we link to this page?

Reply:
@dabit3  I can definitely see the benefits in terms of usability of linking to that page. If we link to the page here, should we include links in the other page references we make in the docs?